src/std/main.ab

```diff
@@ -266,3 +272,15 @@ pub fun shell_var_get(name: Text): Text {
 pub fun shell_unset(name: Text): Null {
     $unset {name}$?
 }
+
+pub fun encode64(text: Text): Text {
+    return unsafe $echo -n "{text}" | base64$
+}
+
+pub fun decodeb64(encoded: Text): Text {
+    return unsafe $echo "{encoded}" | base64 --decode$
+}
+
+pub fun randint(min: Num, max: Num): Num {
+    return unsafe $shuf -i "{min}-{max}" -n 1$
```

Comment:
this is quite controversial. is it widely present on a UNIX system? does its behaviour change from platform to platform? how often is it maintained? does it generate secure random numbers? does it work with floats?

Reply:
also this very obviously will break with negative numbers

Reply:
maybe ``$RANDOM`` with seed is better